libs/watt/src/lib/components/data/watt-data-table.component.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, ContentChild, Input, ViewEncapsulation, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import {
+  VaterFlexComponent,
+  VaterSpacerComponent,
+  VaterStackComponent,
+  VaterUtilityDirective,
+} from '../vater';
+
+import { WattCardComponent } from '../card';
+import { WattSearchComponent } from '../search';
+import { WattTableComponent } from '../table';
+import { WattPaginatorComponent } from '../paginator';
+import { WattEmptyStateComponent } from '../empty-state';
+
+import { WattDataIntlService } from './watt-data-intl.service';
+
+@Component({
+  selector: 'watt-data-table',
+  standalone: true,
+  imports: [
+    CommonModule,
+    VaterFlexComponent,
+    VaterSpacerComponent,
+    VaterStackComponent,
+    VaterUtilityDirective,
+    WattCardComponent,
+    WattEmptyStateComponent,
+    WattPaginatorComponent,
+    WattSearchComponent,
+  ],
+  encapsulation: ViewEncapsulation.None,
+  styles: [
+    `
+      watt-data-table h3,
+      watt-data-table h4 {
+        line-height: 44px;
+        min-height: 44px;
+        margin: 0;
+      }
+
+      watt-data-table watt-data-filters {
+        min-height: 44px;
+      }
+
+      watt-data-table watt-paginator {
+        display: block;
+        margin: calc(-1 * var(--watt-space-m)) -24px -24px;
+      }
+
+      .watt-data-table--empty-state {
+        margin-bottom: var(--watt-space-m);
+        overflow: auto;
+
+        & > watt-empty-state {
+          margin: auto;
+        }
+      }
+    `,
+  ],
+  template: `
+    <watt-card vater fill="vertical">
+      <vater-flex fill="vertical" gap="m">
+        <vater-stack direction="row" gap="m">
+          <vater-stack direction="row" gap="s">
+            <ng-content select="h3" />
```

Comment:
Question: Does forcing either `h3` or `h4` make sense? Couldn't we have cases where the semantic leveling would be `h2`?

Reply:
Good question 🤔 In the DataHub app, the heading at the top of the page is an `h2`, which is why I thought it could never be `h1` or `h2`. But that is of course specific to DataHub. Do you have a use case for `h2` (or even `h5` / `h6`) for EO? I don't mind adding other levels, but perhaps that is also something we could add when the need arises 🙂